### PR TITLE
i#1686 decoder tests: Add some missing OP_vbic_{i16,i32} encodings.

### PR DESCRIPTION
**Pull request by egrimley**
Replace some "INVALID" lines in A32_ext_simd8.
Also correct the opcodes in A32_ext_bit19.

Change-Id: I5fcf10fd028da215730a4f2327a317ab1f2a9e98

### CODE REVIEW

**Comment by derekbruening** [comment]
If you're working on the missing opcodes for #2465, **please assign to yourself to avoid duplicate work** -- I was planning to fill in the missing decode table entries and just now was going to assign to me.

**Comment by egrimley** [comment]
Are any other encodings known to be missing?

Probably i#2465 can be closed after this, though there are a couple of other matters arising there which might be worth creating an issue for, depending on what you and other people think: DEBUG=intermediate and non-DEBUG ASSERTs/printfs.

**Comment by derekbruening** [comment]
Please reference #2465 here

**Comment by egrimley** [comment]
Shall I just add the following paragraph to the log message?

This fixes the primary cause of the crash reported in i#2465.

Or shall I change the issue number and add "Fixes #2565"?

**Comment by egrimley** [comment]
(That would be "i#2465 A32 decoder: Add some missing ..." in the latter case.)

**Comment by derekbruening** [comment]
I would put it in the title: "i#2465, i#1686: ..." or even demote #1686 to just a xref in the body.  This patch is coming directly from #2465.  As to whether it fixes #2465 it depends on whether the other issues are separated out or not.

**Comment by derekbruening** [comment]
I assume you are also fixing the internal consistency error and have a separate patch for that?  (It is hard to tell as not everything seems to be posted in the issue.)

**Comment by egrimley** [comment]
I don't think I've considered any internal consistency error.

**Comment by egrimley** [comment]
Proposed new log message:
```
i#2465 A32 decoder: Add some missing OP_vbic_{i16,i32} encodings.

Replace some "INVALID" lines in A32_ext_simd8.
Also correct the opcodes in A32_ext_bit19.

Xref #1686.

Fixes #2465
```

**Comment by derekbruening** [comment]
Without fixing the internal consistency it doesn't "fixes #2465": it only fixes 1 of the 6 issues.  I guess 4 of them are covered by other issues but the internal consistency remains.

**Comment by egrimley** [comment]
I think the "Encode/decode inconsistency" would be a separate pull request. It doesn't involve SIMD, I don't know whether it involves the decoder, and I don't yet have the test case to reproduce it.

**Comment by egrimley** [comment]
AppVeyor failure is #2246.

File: core/arch/arm/table_a32_unpred.c [hunk 1]
```diff
@@ -298,11 +298,11 @@ const instr_info_t A32_ext_bit7[][2] = {
  */
 const instr_info_t A32_ext_bit19[][2] = {
   { /* 0 */
-    {EXT_SIMD8,         0xf2800000, "(ext simd8  0)", xx, xx, xx, xx, xx, no, x, 0},
-    {EXT_SIMD5,         0xf2880000, "(ext simd5  0)", xx, xx, xx, xx, xx, no, x, 0},
+    {EXT_SIMD8,         0xf2800010, "(ext simd8  0)", xx, xx, xx, xx, xx, no, x, 0},
```
Comment (derekbruening) [inline core/arch/arm/table_a32_unpred.c]: OK, it looks like all children have bit 4 set, but it doesn't really matter at this split point: just informative, nothing reads it.

**Pull request merged**